modules/cnf-image-based-upgrade-installing-lifecycle-agent-using-cli.adoc

```diff
@@ -5,7 +5,7 @@
 [id="installing-lcao-using-cli_{context}"]
 = Installing the {lcao} by using the CLI
 
-You can use the OpenShift CLI (`oc`) to install the {lcao} from the 4.15 Operator catalog on both the seed and target cluster.
+You can use the OpenShift CLI (`oc`) to install the {lcao}.
```

Comment:
below in the text, the channel for LCA in GA shouldn't be "alpha". CCing @rauhersu to let us know what should be the appropriate value
```
apiVersion: operators.coreos.com/v1alpha1
kind: Subscription
metadata:
  name: openshift-lifecycle-agent-subscription
  namespace: openshift-lifecycle-agent
spec:
  channel: "alpha"     # <- needs to be updated
  name: lifecycle-agent
  source: redhat-operators
sourceNamespace: openshift-marketplace
  ```

Reply:
'stable' will be our channel across all lca releases. See table: https://redhat-internal.slack.com/archives/C05JHD9QYTC/p1715779168648329?thread_ts=1715180505.378469&cid=C05JHD9QYTC